bundle/config/validate/interpolation_in_auth_config.go

```diff
@@ -53,7 +60,10 @@ func (f *noInterpolationInAuthConfig) Apply(ctx context.Context, b *bundle.Bundl
 			continue
 		}
 
-		vv := v.MustString()
+		vv, ok := v.AsString()
```

Comment:
You can omit the check for invalid or nil above now.

Reply:
Good point, removed.